creme/creme_api/api/contenttypes/viewsets.py

```diff
@@ -0,0 +1,22 @@
+from rest_framework import viewsets
+from rest_framework.schemas.openapi import AutoSchema
+
+from .serializers import ContentTypeSerializer
+from .utils import get_cremeentity_contenttype_queryset
+
+
+class ContentTypeViewSet(viewsets.ReadOnlyModelViewSet):
+    """
+    retrieve:
+    Retrieve a content type.
+
+    list:
+    List content types.
+    """
+
+    queryset = None
+    serializer_class = ContentTypeSerializer
+    schema = AutoSchema(tags=["Content Types"])
+
+    def get_queryset(self):
+        return get_cremeentity_contenttype_queryset()
```

Comment:
- Is it possible to return a list not based on a Queryset ? (if yes, remove 'get_cremeentity_contenttype_queryset()' )
- Rename EntityContentTypesViewSet VS return all ContentTypes (or entity types + types registered on crme_config + some types in an attributes with Relation/EntityFilters/...). I don't know where this view is used so I ignore what's the smartest.